packages/events-derive/src/lib.rs

```diff
@@ -41,32 +40,34 @@ pub fn try_from(arg: TokenStream, input: TokenStream) -> TokenStream {
         #input
 
         use error_stack::{IntoReport as _, ResultExt as _};
+        use events as _internal_events;
+        use core::convert::TryFrom as _internal_TryFrom;
```

Comment:
Why `_internal_TryFrom`?

Reply:
to get around unexpected behaviour. For example, if the file you use that macro in already imports `TryFrom`, suddenly it wouldn't compile because of a duplicate import. Chances of a name collision with `_internal_TryFrom` are next to zero, so that should avoid this